scripts/ipfs-add-pinlist.py

```diff
@@ -16,10 +16,13 @@ def add_pins(pinlist):
     for pin in pinlist:
         cid = pin["cid"]
 
-        if (name := pin.get("name")) is not None:
-            subprocess.run(["ipfs", "pin", "add", "-r", cid, "-n", name])
+        if (prev := pin.get("prev")) is not None:
+            subprocess.run(["ipfs", "pin", "update", prev, cid])
```

Comment:
I'd add the option not to unpin earlier versions here, as otherwise we'll likely re-pin an unpin things in the pinlist on and on again.

I also don't know how `update` behaves, if the old CID isn't yet pinned. In case it fails, we might have to fall back to add.